angular/projects/spark-angular/src/lib/components/sprk-accordion-item/sprk-accordion-item.component.ts

```diff
@@ -291,7 +288,8 @@ export class SprkAccordionItemComponent implements OnInit {
     const classArray: string[] = [
       'sprk-c-Icon--filled-current-color',
       'sprk-c-Icon--xl',
-      'sprk-u-mrs',
+      'sprk-c-Accordion__icon',
+      'sprk-c-Accordion__icon--leading',
```

Comment:
my first instinct was to adding into the __icon class -- the users don't have access to removing this, so if it would be us engineers that would add or remove this class or margin from style. I'm curious the through process for creating a --leading class

Reply:
We chatted about this one in standup. About it being a difference btw a  leading icon on the left and and icon on the right. Resolving this thread. Woot woot!